cli/util.go

```diff
@@ -15,6 +15,11 @@ type apiPort struct {
 	Port   string
 }
 
+type Volumes struct {
```

Comment:
Our current project/package setup is really suboptimal, but maybe this would rather fit into `cli/volume.go`? :thinking: